server.js

```diff
@@ -1,8 +1,12 @@
 const express = require("express");
+const cors = require("cors");
 const path = require("path");
 const dotenv = require("dotenv");
 const morgan = require("morgan");
 const { uuid } = require("uuidv4");
+const crypto = require("crypto-js");
+const NodeRSA = require("node-rsa");
```

Comment:
@longxice naming style: why not `nodeRSA` ? Or shorter, `rsa`?

Reply:
@anhldbk I think it should be 'rsa', because this is a Node.js application, so it should not include 'node' or any JavaScript-related words.

Reply:
@longxice ok. Go ahead.